lib/rubocop/cop/lint/disjunctive_assignment_in_constructor.rb

```diff
@@ -34,12 +34,13 @@ def on_def(node)
         def check(node)
           return unless node.method_name == :initialize
 
-          check_body(node)
+          check_body(node.body)
         end
 
         # @param [DefNode] node a constructor definition
-        def check_body(node)
-          body = node.body
+        def check_body(body)
```

Comment:
👍 I think it makes good sense that just as `#check` takes a node and `#check_body_lines` takes some lines, `#check_body` takes a body instead of a node.